src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.scss

```diff
@@ -36,32 +35,35 @@ $shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
 .Contents {
   display: flex;
   flex: 1 1 auto;
+  flex-wrap: wrap;
   align-items: center;
   justify-content: space-between;
+  min-height: top-bar-height();
   min-width: 1px;
   max-width: layout-width(primary, max) + layout-width(secondary, max) +
     layout-width(inner-spacing);
   height: 100%;
   margin: 0 auto;
-  padding: 0 spacing();
+  padding: 0 spacing() 1rem;
 
   @include page-content-when-not-fully-condensed {
-    padding: 0 spacing(loose);
+    padding: 0 spacing(loose) 1rem;
   }
 
   @include page-content-when-not-partially-condensed {
-    padding: 0 spacing(extra-loose);
+    padding: 0 spacing(extra-loose) 1rem;
+  }
+
+  > * {
+    margin-top: 1rem;
   }
 }
 
 .Message {
-  @include truncate;
```

Comment:
Removed the truncation of the message in favor of normal text wrapping since the bar itself now supports it.